### DISCUSSION TITLE
Xion - Nights - EDA

### DISCUSSION THREAD

**Discussion by rastitiratnasari** [teamalgoritma/community#817] [Q&A] [answered]
1. Penjelasan dan contoh penggunaan drop.duplcated dengan menggunakan parameter ( keep = first/last)
2. Cara menampilkan data frame yang memiliki missing value nya saja
3. Penggunaan parameter dayfirst=True
4. Cara menambahkan sort_value, ascending dan conditional subseting saat menggunakan method croostab/pivot table. Terkadang masih suka error.

**Comment by arkanayudhistira** [comment, marked as answer]
Dear @rastitiratnasari,

Berikut jawaban untuk pertanyaan-pertanyaan tersebut:

1. Parameter `keep=first` dan `keep=first` pada `drop_duplicates()` mengatur data mana yang akan dipertahankan dari kumpulan data duplikat. Jika `keep=first`, maka yang akan dipertahankan adalah data pertama, jika `keep=last` maka yang akan dipertahankan adalah data terakhir
2. Untuk menampilkan dataframe yang memiliki missing value, anda dapat menggunakan code berikut:

    ```
    household_missing[ household_missing.isna().any(axis=1) ]
    ```
4. Parameter `dayfirst=True` berfungsi untuk menentukan format datetime yang ingin dikonversi adalah data dengan format `DD-MM-YYYY` (ex: 10-11-2023 agar dibaca sebagai 10 November 2023, bukan 11 Oktober 2023)
5. Method `sort_values()` dapat diberikan input berupa nama kolom dalam string seperti contoh `sort_values('Total')` sesuai dengan nama kolom pada dataframe

Semoga membantu!

**Comment by rastitiratnasari** [comment]
Dear Mas Arkana,

Well received.
Terimakasih atas support nya.
